src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -131,6 +131,22 @@ protected override void ProcessRecord()
                             }
                         }
 
+                        //RootModule can be null, empty string or point to a valid .psm1, or .dll.  Anything else is invalid.
```

Comment:
This file contains `OrdinalIgnoreCase` 4 times. But I see lots of other files to ignore the case when dealing with paths.